### PR TITLE
14 add preload world results param

### PR DESCRIPTION
**Pull request by janeeisenstein**
Added `loadWorldResults` parameter that can be used to prevent run world results from being loaded into the simpl state.

### CODE REVIEW

File: src/decorators/simpl.js [hunk 1]
```diff
@@ -71,9 +74,14 @@ export function simpl(options) {
     } else {
       optionsWithDefaults.loadAllScenarios = false;
     }
+    if (options.hasOwnProperty('loadWorldResults')) {
+      optionsWithDefaults.loadWorldResults = options.loadWorldResults;
+    } else {
+      optionsWithDefaults.loadWorldResults = true;
```
Comment (fcurella) [inline src/decorators/simpl.js]: shouldn't we add the defaults values in `utils. wampOptionsWithDefaults()`?

Comment (janeeisenstein) [reply src/decorators/simpl.js]: Why? const optionsWithDefaults = wampOptionsWithDefaults(options);

Comment (janeeisenstein) [reply src/decorators/simpl.js]: This option is only used in the simpl decorator.

Comment (fcurella) [reply src/decorators/simpl.js]: good point

**Pull request merged**